ebcli/bundled/asciimatics/exceptions.py

```diff
@@ -14,10 +14,6 @@
 """
 This module defines the exceptions used by asciimatics.
 """
-from __future__ import division
-from __future__ import absolute_import
-from __future__ import print_function
-from __future__ import unicode_literals
```

Comment:
Random unsolicited comments - I just came here to see if this package was going to remove the dependency on `future` because I saw the same CVE. Thanks for sending the PR!

These `__future__` imports come from Python, not the `future` third-party library. ([reference](https://docs.python.org/3/library/__future__.html))

The future module provides symbols at `builtins` which is used in ebcli/bundled/asciimatics/screen.py ([reference](https://github.com/PythonCharmers/python-future))

I think those are the only uses of the `future` library, and the code can just use `ord` `chr` etc directly without an import, assuming there's no longer any support for Python 2.